core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/node-exporter.yml

```diff
@@ -44,6 +45,8 @@
           include_role:
             name: node_exporter
             tasks_from: install-node-exporter-as-daemonset.yml
+          vars:
+            specification: "{{ exporter_vars.specification }}"
```

Comment:
Don't we want to name it as `_specification`? This is a general question for all task vars we define.

Reply:
Yes, we want but not in this PR. This would require additional changes (in `node_exporter` role) and tests for apply mode since `specification` is used there in many places and upgrade mode re-uses code from `node_exporter` role. I believe we can do that when changing `node_exporter` role.